mirrord/layer/src/file/filter/read_local_by_default.rs

```diff
@@ -46,6 +46,11 @@ pub fn regex_set_builder() -> RegexSetBuilder {
         // TODO: `node` searches for this file in multiple directories, bypassing some of our
         // ignore regexes, maybe other "project runners" will do the same.
         r"^.*/package.json$",
+        r"^.*/.yarnrc$",
+        r"^.*/.yarnrc.yml$",
+        r"^.*/.yarnrc.json$",
+        r"^.*/node_modules",
```

Comment:
Nitpick: `^.*` is superfluous. We can leave it there thought if you think it's more readable or something.

```suggestion
        r"/\.yarnrc$",
        r"/\.yarnrc.yml$",
        r"/\.yarnrc.json$",
        r"/node_modules",
```

Reply:
And is it on purpose that the `node_modules` one does not end with a `$`?

Reply:
yeah I want to get it on any path